scripts/get_ps_list.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+import sys
+import argparse
+import os
+import time
+import ififuncs
+import csv
+
+def parse_args(args_):
+    parser = argparse.ArgumentParser(
+        description='Get a CSV of AIPs in preservation storage with accession number and object entry number'
+					' Written by Yazhou He.'
+    )
+    parser.add_argument(
+        '-i',
+        help='root path of preservation_storage', required=True
+    )
+    parsed_args = parser.parse_args(args_)
+    return parsed_args
+
+def main(args_):
```

Comment:
it looks like this whole function is using tabs? Did you run this script against pylint? It probably would have flagged that there was a mix of tabs and spaces.

Reply:
Awful... must happened between different workspaces. fixed.